api/src/main/java/org/openmrs/PersonAttributeType.java

```diff
@@ -9,31 +9,50 @@
  */
 package org.openmrs;
 
-import java.io.Serializable;
-import java.util.Comparator;
-
 import org.codehaus.jackson.annotate.JsonIgnore;
 import org.hibernate.search.annotations.Field;
 import org.openmrs.util.OpenmrsUtil;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.Comparator;
+
 /**
  * PersonAttributeType
  */
+@Entity
+@Table(name = "person_attribute_type")
 public class PersonAttributeType extends BaseChangeableOpenmrsMetadata implements java.io.Serializable, Comparable<PersonAttributeType> {
 	
 	public static final long serialVersionUID = 2112313431211L;
 	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "person_attribute_type_id")
```

Comment:
How are we capturing this? `unsaved-value="0"`

Reply:
So my thinking was that in the JPA annotation mapping, we don't need to explicitly set unsaved-value="0", as this behavior is already handled by the GenerationType.IDENTITY strategy. But let me add the `nullable=false`